ui/lib/core/addon/templates/components/replication-action-reindex.hbs

```diff
@@ -28,25 +26,28 @@
 >
   <section class="modal-card-body">
     <p class="has-bottom-margin-m">
-      Reindexing can cause a very long delay on the number and size of objects in the data store. {{if model.replicationAttrs.isPrimary 'You should always re-index your secondary first.'}}
+      Reindexing can cause a very long delay depending on the number and size of objects in the data store.
+      {{if model.replicationAttrs.isPrimary 'You should always re-index your secondary first.'}}
     </p>
     <p>
-      Progress will be shown, and you will {{ if model.replicationAttrs.isPrimary 'not' }} be able to use Vault during this time.
+      Progress will be shown, and you will
+      {{if model.replicationAttrs.isPrimary 'not'}}
```

Comment:
Let's make sure the formatting here didn't remove all the spaces around `not` so it doesn't look like `you willnotbe able...`

Reply:
We can just add the spaces around "not" if it did break the spaces 👍

Reply:
you know, that's interesting -- i totally would've thought that this would smoosh the words together too, but it didn't 🤔 

![image](https://user-images.githubusercontent.com/903288/83695428-dedadf80-a5ae-11ea-98d8-002ba8dc4bed.png)